### PR TITLE
GCP check ssh auth

### PR DESCRIPTION
**Pull request by jonathan-greig**
Relates to #338

### CODE REVIEW

File: libcloudforensics/providers/gcp/forensics.py [hunk 1]
```diff
@@ -430,3 +432,63 @@ def VMRemoveServiceAccount(project_id: str,
...
+
+  project = gcp_project.GoogleCloudProject(project_id)
+  instance = project.compute.GetInstance(instance_name)
+  instance_info = instance.GetOperation()
+  networks = instance_info.get('networkInterfaces', [])
+
+  external_ips = []
+  for network in networks:
+    if 'accessConfigs' in network:
+      for config in network['accessConfigs']:
```
Comment (sa3eed3ed) [inline libcloudforensics/providers/gcp/forensics.py:478]:
```suggestion
    for config in network.get('accessConfigs'):
```
maybe you can use get inn the loop and get rid of the if condition before

**Review by sa3eed3ed — CHANGES_REQUESTED** [review]
Thank you for working on this, I like the feature.
just few small comment.

File: libcloudforensics/providers/gcp/forensics.py (hunk 1 above)
Comment (jonathan-greig) [reply libcloudforensics/providers/gcp/forensics.py:478]: Nice! That is more concise, change made under the new function in GoogleCloudCompute.

File: libcloudforensics/providers/gcp/forensics.py [hunk 2]
```diff
@@ -430,3 +432,63 @@ def VMRemoveServiceAccount(project_id: str,
...
+        if 'natIP' in config:
+          external_ips.append(config['natIP'])
```
[the remark this answers is not in the document]
Comment (jonathan-greig) [reply libcloudforensics/providers/gcp/forensics.py:480]: Moved the functionality to a new GetNatIps method in the GoogleCloudCompute class.

**Review by sa3eed3ed — APPROVED** [review]
LGTM

**Pull request merged**